Lib/test/test_xml_etree.py

```diff
@@ -1044,6 +1044,31 @@ def test_html_empty_elems_serialization(self):
                                        method='html')
                 self.assertEqual(serialized, expected)
 
+    def test_dump_attribute_order(self):
+        # See BPO 34160
+        e = ET.Element('cirriculum', status='public', company='example')
+        with support.captured_stdout() as stdout:
+            ET.dump(e)
+        self.assertEqual(stdout.getvalue(),
+                         '<cirriculum status="public" company="example" />\n')
+
+    def test_tree_write_attribute_order(self):
+        # See BPO 34160
+        root = ET.Element('cirriculum', status='public', company='example')
+        tree = ET.ElementTree(root)
+        try:
+            fo = open(support.TESTFN, "wb")
```

Comment:
Not sure what *serialize()* is.  Switched from the traditional test support code to a context manager that does make it a little nicer.

Reply:
`serialize()` is a helper in this file which writes an XML to string/bytes/StringIO/BytesIO. It will allow you to use a single line instead of three lines.
```python
self.assertEqual(serialize(tree),
                 '<cirriculum status="public" company="example" />')
```